.github/workflows/publish-on-release.yml

```diff
@@ -27,15 +30,15 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
         uses: mikepenz/release-changelog-builder-action@v5
         with:
-          configuration: ./.github/json/config.json
-          toTag: ${{ github.event.release.tag }}
+          configuration: './.github/json/config${{ steps.tag.outputs.O }}.json'
           ignorePreReleases: true
           fetchViaCommits: true
           failOnError: true
 
   publish-20:
     name: 1.20.1
     needs: [ meta ]
+    if: ${{ contains(github.ref_name, '1.20.1') || !contains(github.ref_name, '1.21') }}
```

Comment:
I'm not 100% sure for this (and the next change), but `&&` might be better here, instead of `||`